### PR TITLE
Convert git history

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

File: src/repos-to-monorepo.ts [hunk 1]
```diff
@@ -7,12 +7,13 @@ import { createLibrary } from './lib/create-library';
...
 import { modifyBasePackageJson } from './lib/modify-base-package-json';
 import { modifyBaseTsConfig } from './lib/modify-base-tsconfig';
+import { gitAdd, gitCommit } from './utils/clone-repo';
 import { readJsonC, writeJsonC } from './utils/jsonc-utils';
 import { runCommand } from './utils/run-command';
 import { sortObjectByKeys } from './utils/sort-object-by-keys';
 
 const CWD = process.cwd();
-const TEMP_PATH = path.join(CWD, '.skyuxtemp');
+const TEMP_PATH = path.join(CWD, '..', '.skyuxtemp');
```
Comment (johnhwhite) [inline src/repos-to-monorepo.ts:16]: I move the temp directory up a level so it doesn't conflict with the git repo for this project.

Comment (Blackbaud-SteveBrush) [reply src/repos-to-monorepo.ts:16]: Good idea.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
- For the code examples, it looks like the changes moved the file to `apps/code-examples/src/app/code-examples/forms/code-examples/checkbox/basic` instead of `apps/code-examples/src/app/code-examples/forms/checkbox/basic`. Possible to fix with the git tools?
- Also, the module that is autogenerated by `nx g` is persisting and should be deleted. e.g. `libs/avatar/src/lib/avatar.module.ts` should be deleted.

**Comment by johnhwhite** [comment]
> * For the code examples, it looks like the changes moved the file to `apps/code-examples/src/app/code-examples/forms/code-examples/checkbox/basic` instead of `apps/code-examples/src/app/code-examples/forms/checkbox/basic`. Possible to fix with the git tools?
> * Also, the module that is autogenerated by `nx g` is persisting and should be deleted. e.g. `libs/avatar/src/lib/avatar.module.ts` should be deleted.

These issues should now be addressed.

**Pull request merged**